src/ocm/components/featureSupportLevels/ReviewClusterFeatureSupportLevels.tsx

```diff
@@ -113,7 +114,7 @@ export const ClusterFeatureSupportLevelsDetailItem: React.FC<{ cluster: Cluster
   const featureSupportLevelData = useFeatureSupportLevel();
 
   const clusterFeatureSupportLevels = React.useMemo(() => {
-    return getLimitedFeatureSupportLevels(cluster, featureSupportLevelData);
+    return getLimitedFeatureSupportLevels(cluster, featureSupportLevelData, sentryErrorHandler);
```

Comment:
Why not just call `captureException`?

```suggestion
    return getLimitedFeatureSupportLevels(cluster, featureSupportLevelData, captureException);
```